app/flatpak-cli-transaction.c

```diff
@@ -293,7 +293,10 @@ progress_changed_cb (FlatpakTransactionProgress *progress,
           max = flatpak_transaction_operation_get_download_size (op);
           formatted_max = g_format_size (max);
           transferred = flatpak_transaction_progress_get_bytes_transferred (progress);
-          formatted = g_format_size (transferred);
+          if (transferred < 1024) // avoid "bytes"
+            formatted = g_strdup ("0.0 kB");
```

Comment:
Showing 0.0 seems wrong? Round up to 1.0?

Reply:
yeah, 1.0 is better. will change that